refiner-lib/src/refiner_inner.rs

```diff
@@ -1250,6 +1266,8 @@ enum RefinerError {
     FailNearTx,
     /// Could not get data from the promise result
     PromiseResultError,
+    /// The error while building Aurora transaction in the scope of the build_transaction function
+    BuildAuroraTransactionError,
```

Comment:
```suggestion
    BuildAuroraTransactionError(String),
```
It would be more informative.